atomic_reactor/util.py

```diff
@@ -597,13 +597,16 @@ def get_build_json():
         raise
 
 
-def is_scratch_build():
-    build_json = get_build_json()
-    try:
-        return build_json['metadata']['labels'].get('scratch', False)
-    except KeyError:
-        logger.error('metadata.labels not found in build json')
-        raise
+def get_build_json_metadata(workflow):
```

Comment:
get_build_json is used by inputplugin to obtain input for workflow so at that time workflow doesn't even exist